net-p2p/syncthing/syncthing-0.14.12.ebuild

```diff
@@ -17,6 +17,8 @@ SLOT="0"
 KEYWORDS="~amd64 ~x86 ~arm"
 IUSE="tools"
 
+DEPEND="tools? ( >=dev-lang/go-1.7.1:= )"
```

Comment:
Hi All,

syncthing comes optionally with a set of utilities
```
$ ls /usr/libexec/syncthing/
stbench     stdiscosrv  stfinddevice  strelaypoolsrv  stvanity
stcompdirs  stevents    stgenfiles    strelaysrv      stwatchfile
stdisco     stfileinfo  stindex       stsigtool       testutil
```
which we currently install if `USE="tools"`. These utilities include in particular the global discovery server, the relay server, the relay pool server, the benchmark utility, and the testing utility.

It you don't like calling this use flag `tools`, there are options: rename is to `utils` or `extra` for example. Or introduce separate use flags for all the extra tools (which would make it hard to maintain). Or introduce separate packages in portage for the tools. Or something else.

Personally, I find `tools` a suitable name for this use flag.